data/pc/1.6.4/protocol.json

```diff
@@ -0,0 +1,2286 @@
+          {
+            "name": "byte2",
+            "type": "i8"
+          },
+          {
+            "name": "blockId",
+            "type": "i16"
+          }
+        ]
+      ],
+      "packet_block_break_animation": [
+        "container",
+        [
+          {
+            "name": "entityId",
+            "type": "i32"
+          },
+          {
+            "name": "position",
+            "type": "position_iii"
+          },
+          {
+            "name": "destroyStage",
+            "type": "i8"
+          }
+        ]
+      ],
+      "packet_map_chunk_bulk": [
+        "container",
+        [
+          {
+            "name": "chunkColumnCount",
+            "type": [
+              "count",
+              {
+                "type": "i16",
+                "countFor": "meta"
+              }
+            ]
+          },
+          {
+            "name": "dataLength",
+            "type": [
+              "count",
+              {
+                "type": "i32",
+                "countFor": "compressedChunkData"
+              }
+            ]
+          },
+          {
+            "name": "skyLightSent",
+            "type": "bool"
+          },
+          {
+            "name": "compressedChunkData",
+            "type": [
+              "buffer",
+              {
+                "count": "dataLength"
+              }
+            ]
+          },
+          {
+            "name": "meta",
+            "type": [
+              "array",
+              {
+                "count": "chunkColumnCount",
+                "type": [
+                  "container",
+                  [
+                    {
+                      "name": "x",
+                      "type": "i32"
+                    },
+                    {
+                      "name": "z",
+                      "type": "i32"
+                    },
+                    {
+                      "name": "bitMap",
+                      "type": "u16"
+                    },
+                    {
+                      "name": "addBitMap",
+                      "type": "u16"
+                    }
+                  ]
+                ]
+              }
+            ]
+          }
+        ]
+      ],
+      "packet_explosion": [
+        "container",
+        [
+          {
+            "name": "x",
+            "type": "f64"
+          },
+          {
+            "name": "y",
+            "type": "f64"
+          },
+          {
+            "name": "z",
+            "type": "f64"
+          },
+          {
+            "name": "radius",
+            "type": "f32"
+          },
+          {
+            "name": "recordCount",
+            "type": [
+              "count",
+              {
+                "type": "i16",
+                "countFor": "records"
+              }
+            ]
+          },
+          {
+            "name": "records",
+            "type": [
+              "array",
+              {
+                "count": "recordCount",
```

Comment:
This needs to be looked at, as in 1.6.4 these records are just positions relative to the explosion written as bytes. 

See https://wiki.vg/index.php?title=Protocol&oldid=4899#Explosion_.280x3C.29